editor/scene_tree_dock.cpp

```diff
@@ -948,6 +948,18 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 				break;
 			}
 
+			if (tocopy->get_owner() != scene) {
+				accept->set_text(TTR("Can't sSavave a branch which is a child of an already instantiated scene.\nTo save this branch into its own scene, open the original scene, right click on this branch, and select \"Save Branch as Scene\"."));
```

Comment:
For the record, I fixed up this typo in 02d9ac107158c8c5d95f1ecc48078d66e00c1a57 (and in the `3.4` cherry-pick of this PR).

Reply:
I...honestly don't know how that slipped through 👀